### ISSUE TITLE
WHITE is looks green...

### ISSUE THREAD

**Issue by yasushisakai** [nannou-org/nannou#740]
Hi I've been testing out but, seems that the `WHITE`or `Rgb(1.0, 1.0, 1.0)` appears to be green. Screenshot attached.

```rust
use nannou::prelude::*;
struct Model {}
fn main() {
    nannou::app(model).update(update).simple_window(view).run();
}
fn model(_app: &App) -> Model {
    Model {}
}
fn update(_app: &App, _model: &mut Model, _update: Update) {}
fn view(app: &App, _model: &Model, frame: Frame) {
    let draw = app.draw();
    let white = Rgb::new(1.0, 1.0, 1.0);
    draw.ellipse().x_y(-60.0, 0.0).w_h(50.0, 50.0).color(WHITE);
    draw.ellipse().x_y(60.0, 0.0).w_h(50.0, 50.0).color(white);
    draw.to_frame(app, &frame).unwrap();
}
```
![Screen Shot 2021-05-17 at 6 32 21 PM](https://user-images.githubusercontent.com/1502180/118564418-3a5d9900-b73e-11eb-97c0-5727a47d30de.png)

I'm using Mac 11.2.3 (M1), rustc 1.50.0, nannaou 0.16.0.

**Comment by yasushisakai** [comment]
I've run the same program with an Intel Mac can confirm that it renders with the correct color.

**Comment by danwilhelm** [comment]
It seems to render correctly (as white) on Windows too.

**Comment by j6k4m8** [comment]
@yasushisakai  — do you have a funky monitor setup? when I click and drag a sketch from one monitor to another everything goes blueish. i wonder if this is related... (Renders correctly on the primary monitor!)

**Comment by yasushisakai** [comment]
@j6k4m8 unfortunately, I'm using the built-in display... and It won't change if I use a different one..

**Comment by yasushisakai** [comment]
Assuming this is a lower-level issue, and feel a bit guilty asking it here... but are there pointers that maybe I can investigate deeper?

**Comment by JoshuaBatty** [comment]
Can you try running the examples in `[wpgu-rs](https://github.com/gfx-rs/wgpu-rs)` and report if you are getting strange color behavior? If so, then it's a `wgpu` bug. Would be a good place to start

**Comment by yasushisakai** [comment]
apologies for keeping this dangling.

I managed to run examples in the `wpgpu-rs`'s branch `0.7` and `0.8`, namely `hello-triangle`, `hello` and `cube` but didn't see anything that gives clues to this issue. Everything worked fine.

**Comment by JoshuaBatty** [comment]
Ok, thanks for reporting. Now that wgpu `0.8` has been merged into master, would you mind pulling down the latest master branch of nannou and seeing if the problem still persists?

**Comment by DingDean** [comment]
I'm also using an M1 mac mini, the problem still persists when on the latest master branch with wgpu updated to 0.8.

I wrote a simple program with wgpu-rs 0.7 directly and the color is normal there. It does not seem to be a problem with wgpu-rs.

**Comment by danwilhelm** [comment]
@DingDean Perhaps your monitor resolutions are different bit depths, eg the primary is 10-bit color and the secondary 8-bit? Perhaps Nannou detects the 10-bit and initializes a 10-bit framebuffer which isn't supported or converted correctly when it's extended to the other display?

**Comment by DingDean** [comment]
> @DingDean Perhaps your monitor resolutions are different bit depths, eg the primary is 10-bit color and the secondary 8-bit? Perhaps Nannou detects the 10-bit and initializes a 10-bit framebuffer which isn't supported or converted correctly when it's extended to the other display?

I just double checked my only monitor which is ["DisplayHDR 400" verified](https://displayhdr.org/). According to the verification, the display is 8-bit color depth.

Also, I just ran all the examples in the nannou repo and everything is fine. Yet the simplest "rect.color(BLUE)" just doesn't work, really weird.

**Comment by danwilhelm** [comment]
It seems that the example `draw_text` draws some blue text. Perhaps you can also try `draw.background().color(BLUE);` to set the background blue. I wonder if these also exhibit the color difference?

**Comment by DingDean** [comment]
> It seems that the example `draw_text` draws some blue text. Perhaps you can also try `draw.background().color(BLUE);` to set the background blue. I wonder if these also exhibit the color difference?

Wow, I set the background color to white and then the color display returns to normal. I'll attached the screenshot below:

*Program without setting background color*:
<img width="1589" alt="截屏2021-06-12 下午6 33 20" src="https://user-images.githubusercontent.com/11747908/121773223-05adf780-cbad-11eb-9816-109d3a7ee7cb.png">

*Program with background color*:
<img width="1585" alt="截屏2021-06-12 下午6 33 44" src="https://user-images.githubusercontent.com/11747908/121773231-11012300-cbad-11eb-99f0-94b80522bc7f.png">

**Comment by danwilhelm** [comment]
Briefly looking at the Nannou source code, it appears that if the background isn't filled in `view` then Nannou tries to load the prior contents (`LoadOp::Load`). However, the wgpu docs say "The render target must be cleared at least once before its content is loaded." 

So here, I'd guess you are seeing prior contents or uninitialized contents that apparently have some sort of color blending applied, hence the strange color.

This is probably considered a bug - perhaps Nannou should always clear the buffer on the first frame even if the user doesn't specifically request a clear!

**Comment by danwilhelm** [comment]
@yasushisakai — I see that your original code example does not clear the background. Does specifying the background color resolve this for you as well? 

`draw.background().color(BLUE);`

**Comment by yasushisakai** [comment]
@danwilhelm yep, I can confirm that when you clear the background, the color appears to be correct! Leaving the decision to close this issue or to leave this as a bug.

thank you @danwilhelm !

**Comment by danwilhelm** [comment]
Thanks also to @DingDean for reporting and trying some code out!

@j6k4m8  Does clearing the background also resolve the multimonitor bluish tint? If not, it might be helpful to open a separate issue.
